.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,77 @@
+name: Bug report
+description: Create a report to help us improve
+title: 'bug: something is wrong :('
```

```suggestion
title: 'Bug: '
labels: ["bug", "triage"]
```

Created a triage label